cmd.go

```diff
@@ -620,6 +633,9 @@ func (t *ugbt) availableVersions(ctx context.Context, mod string) ([]info, error
 
 			i, err := t.info(ctx, url)
 			if err != nil {
+				if errors.Is(err, statusError{}) {
```

Comment:
The doc spec says errors other than 404 and 410 should abort https://go.dev/doc/go1.13#modules . Maybe differentiate handling there ?

Reply:
Done